src/components/forms/Form/SubmitError.tsx

```diff
@@ -1,26 +1,33 @@
-import { ReactNode, useContext } from 'react';
+import { ReactNode, useContext, isValidElement, memo } from 'react';
 
 import { Alert, CubeAlertProps } from '../../content/Alert';
 
 import { FormContext } from './Form';
 
 type SubmitErrorProps = {
```

Comment:
I think this name isn't good bc it is named like it describes `SubmitError` props, but it is not. I consider renaming it to smth more clear.

My suggestion: 

```typescript
export type SubmitErrorProps = CubeAlertProps

type SubmitErrorContextProps = {
  submitError: unknown;
}
```